lib/Support/CompressionStream.cpp

```diff
@@ -23,17 +23,17 @@
 
 namespace klee {
 
-compressed_fd_ostream::compressed_fd_ostream(const char *Filename,
+compressed_fd_ostream::compressed_fd_ostream(std::string &Filename,
                                              std::string &ErrorInfo)
     : llvm::raw_ostream(), pos(0) {
   ErrorInfo = "";
   // Open file in binary mode
 #if LLVM_VERSION_CODE >= LLVM_VERSION(3, 5)
   std::error_code EC =
-      llvm::sys::fs::openFileForWrite(Filename, FD, llvm::sys::fs::F_None);
+      llvm::sys::fs::openFileForWrite(Filename.c_str(), FD, llvm::sys::fs::F_None);
```

Comment:
Did you try just `Filename` - at least in newer versions there is a Twine ctor for std::string